src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg" alt=""
+        >
+        <h2 class="card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+        <div class="card__code">
+          <span>
+            Код товара: 195434
+          </span>
+        </div>
+        <div class="card__rating">
+          <div class="card__rank stars">
```

Comment:
add stars--4 modifier here, and write styles for all the possible modifiers, so that changing class in html to stars--3 would change number of active stars accordingly